packages/earth-map/src/components/back-to-location/component.tsx

```diff
@@ -9,17 +26,17 @@ interface IProps {
 
 const BackToLocation = ({ location, organization, onClick }: IProps) => {
   const { t } = useTranslation();
+  const classes = useStyles();
 
   return (
-    <div
-      onClick={onClick}
-      className="marapp-qa-backtolocation ng-c-cursor-pointer ng-padding-vertical ng-padding-medium-horizontal ng-ep-background-dark ng-ep-border-top"
-    >
-      <em className="ng-color-white">
-        {t('Return to')} {location}
-        <span className="ng-icon-bullet ng-margin-small-horizontal" />
-        <span className="ng-color-mdgray">{organization}</span>
-      </em>
+    <div className={classes.root}>
+      <Button className="marapp-qa-backtolocation" fullWidth={true} onClick={onClick} size="small">
+        <em>
+          {t('Return to')} {location}
+          <IconCircleSmall />
+          <span>{organization}</span>
```

Comment:
Can we replace these with some <Typography /> components and drop the custom style?